code/Model/SiteTree.php

```diff
@@ -3008,6 +2709,7 @@ public function getTreeTitle()
     {
         $children = $this->creatableChildPages();
         $flags = $this->getStatusFlags();
+        $this->extend('updateStatusFlagsForTreeTitle', $flags);
```

Comment:
This new extension hook allows fluent to exclude its flags from the site tree list, because they look awful there.
In the next set of PRs after this one, it'll be on `CMSMain` instead, but for now `SiteTree` directly handles how things look in that tree.